### PR TITLE
Match func_8004C444

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

File: src/26840_len_20d0.c [hunk 1]
```diff
@@ -197,7 +197,45 @@ INCLUDE_ASM(void, "26840_len_20d0", snd_set_modifiers, SoundManager* manager, So
...
+                    case 0:
+                        func_8004C578(arg0, temp_a1, temp_a2, i);
+                        break;
+                    case 1:
+                        func_8004C884(arg0, temp_a1, temp_a2, i);
+                        break;
+                    case 2: // Yes, this is needed.
+                        break;
+                }
+                continue;
```
Comment (ethteck) [inline src/26840_len_20d0.c:229]: This continue shouldn't be necessary, since it'll do that anyway

File: src/26840_len_20d0.c [hunk 2]
```diff
@@ -197,7 +197,45 @@ INCLUDE_ASM(void, "26840_len_20d0", snd_set_modifiers, SoundManager* manager, So
...
+    u32 phi_a3 = arg0->sfxPlayerSelector;
+    u16 temp = arg0->unk_60;
+    u16 playCounter = arg0->playCounter;
+    SoundPlayer* temp_a1;
+    UnkAl48* temp_a2;
+    u8 i;
+
+    arg0->unk_60 = temp + playCounter;
+
+    for (i = phi_a3, phi_a3 += 8; i < (u8)(phi_a3); i++){
```
Comment (ethteck) [inline src/26840_len_20d0.c:213]: No parens needed around phi_a3

File: src/26840_len_20d0.c [hunk 3]
```diff
@@ -197,7 +197,45 @@ INCLUDE_ASM(void, "26840_len_20d0", snd_set_modifiers, SoundManager* manager, So
...
+        temp_a1 = &arg0->unk_16C[i - arg0->sfxPlayerSelector];
+        if (temp_a1->sefDataReadPos != 0) {
+            arg0->unk4 = temp_a2 = &arg0->soundData->unk_1320[i];
+            if ((arg0->unk4->unk_45 <= arg0->unk_BC)) {
```
Comment (ethteck) [inline src/26840_len_20d0.c:217]: Double parens here

**Review by ethteck — CHANGES_REQUESTED** [review]

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.01%) on `us`.

**Review by ethteck — APPROVED** [review]

**Review by nanaian — APPROVED** [review]

**Pull request merged**